package.json

```diff
@@ -13,9 +13,13 @@
     "lint:fix": "eslint --cache --fix .",
     "postinstall": "lerna bootstrap",
     "snapshot": "node index.js",
```

Comment:
update this too.

Reply:
This is the existing command to generate data file without creating PR. Keeping it as it is.